build.gradle

```diff
@@ -222,7 +222,7 @@ def versions = [
         reformLogging   : '5.1.7',
         sonarPitest     : '0.5',
         springfoxSwagger: '2.8.0',
-        sscsCommon      : '4.9.8',
+        sscsCommon      : '4.9.7-QA-TEST-f098937',
```

Comment:
We can't merge the sscsCommon version change as is. We'll have to make a separate common tag and use it as per the guidelines.